src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -713,6 +721,27 @@ public boolean isAllFeedbackPathOptionsEnabled() {
         return true;
     }
     
+    public boolean isAllRecipientOptionsDisplayed(int questionNumber) {
+        List<WebElement> recipientOptions =
+                browser.driver.findElements(By.cssSelector("#recipienttype-" + questionNumber + " option"));
+        for (WebElement recipientOption : recipientOptions) {
+            String recipientOptionClass = recipientOption.getAttribute("class");
+            if ("hidden".equals(recipientOptionClass)) {
```

Comment:
Yup it doesn't work, it returns `true` regardless. Maybe I could do a verifyHtmlByPart instead?